lib/open-amp/Kconfig

```diff
@@ -18,3 +18,9 @@ config OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF
 	help
 	  This option specifies the number of buffer used in a Vring for
 	  interprocessor communication
+
+config OPENAMP_WITH_DCACHE
+	bool "Build OpenAMP with vrings cache operations enabled"
```

Comment:
Should this config be `default y`? Is there a reason to disable OpenAMP's D$ management while Zephyr's is enabled

Reply:
maybe you have buffers and vrings in a special memory region without a backing cache.

Reply:
So OpenAMP's cache management does not follow memory cacheability configuration?
Maybe it would be safer to use `default y` in such circumstances? I think it's better to do cache management on uncacheable memory (performance impact) than skip cache management on cacheable memory (data corruption)

Reply:
> So OpenAMP's cache management does not follow memory cacheability configuration?

Well, not at least it is configurable.

> Maybe it would be safer to use `default y` in such circumstances? I think it's better to do cache management on uncacheable memory (performance impact) than skip cache management on cacheable memory (data corruption)

I'm always more prone to leave the user to configure this.

Maybe a good compromise could be `default y if CONFIG_CACHE_MANAGEMENT`?

Reply:
Sure, with `depends on CACHCE_MANAGMENT`, the result would be equivalent.

I'm concerned about leaving too much config to users, because I heard user voices they are overwhelmed by configuration complexity in Zephyr. That's why I think it would be perfect if default config worked out-of-the-box with possibility to tweak and optimize by more competent users.

Reply:
ok, no problem. Change incoming.